### PR TITLE
Invertable classification with alpha

### PR DESCRIPTION
**Pull request by bagnell**
Adds `Scene.invertClassification` and `Scene.invertClassificationAlpha`.  When `Scene.invertClassification` is set to `true`, any 3D Tiles geometry that intersects classification volumes will be drawn opaque while geometry that does not intersect the volume will be drawn translucent with the alpha set to `Scene.invertClassificationAlpha`.

I'm open to different names for these properties.

@pjcozzi @lilleyse The derived commands approach doesn't seem very clean. Another option would be to add new passes.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@bagnell thanks for the pull request!

I noticed that [`CHANGES.md`](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the Cesium API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and bump this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

__I am a bot who helps you make Cesium awesome!__ Thanks again.

**Comment by pjcozzi** [comment]
At first glance names seem OK.

Are you sure this should be a scene-level setting?  Perhaps we need a way to composite a primitives/tilesets into a micro pass that has these settings (which also answers your derived commands question and might also be able to fix #5676)?

File: Apps/Sandcastle/gallery/3D Tiles Classification.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'https://beta.cesium.com/api/assets/1458?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIxYmJiNTAxOC1lOTg5LTQzN2EtODg1OC0zMWJjM2IxNGNlYmMiLCJpZCI6NDQsImFzc2V0cyI6WzE0NThdLCJpYXQiOjE0OTkyNjM4MjB9.1WKijRa-ILkmG6utrhDWX6rDgasjD7dZv-G5ZyCmkKg'
+}));
+
+var classification = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8002/tilesets/meshTest/'
```
Comment (pjcozzi) [inline Apps/Sandcastle/gallery/3D Tiles Classification.html]: Can you put this - or something like this - on beta.cesium.com?

File: Source/Scene/FrameState.js [hunk 2]
```diff
@@ -296,6 +296,20 @@ define([
...
+         * @type {Boolean}
+         * @default false
+         */
+        this.invertClassification = false;
+
+        /**
+         * The alpha of unclassified 3D Tile geometry when {@link FrameState#invertClassification} is <code>true</code>.
+         * @type {Number}
+         */
+        this.invertClassificationAlpha = undefined;
```
Comment (pjcozzi) [inline Source/Scene/FrameState.js]: Should this default to `1.0` instead?

Comment (pjcozzi) [inline Source/Scene/FrameState.js]: Or maybe not since only the `Scene` default matters...

**Comment by bagnell** [comment]
I'm merging this. What needs reviewing is now in #5836.

**Pull request merged**